src/App.js

```diff
@@ -2,16 +2,23 @@
 /* eslint-disable no-unused-vars */
 
```

Comment:
- [ ] Please, consider removing all linter disables. Let linter makes its direct job - help you find bugs on early stages of building the app 👍